services/api-server/setup.py

```diff
@@ -63,4 +64,9 @@ def read_reqs(reqs_path: Path) -> Set[str]:
         test_suite="tests",
         tests_require=TEST_REQUIREMENTS,
         extras_require={"test": TEST_REQUIREMENTS},
+        entry_points={
+            "console_scripts": [
+                "simcore-service-api-server = simcore_service_api_server.cli:main",
```

Comment:
ok that is what you want me to add in the catalog right?

Reply:
that was my suggestion, yes :-)